index.html

```diff
@@ -0,0 +1,1249 @@
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                </div>
+            </section>
+
+            <!-- Modal -->
+
+            <!-- Ratings -->
+
+            <section id="rating">
+                <div class="code-container">
+                    <h2 class="component-title">Ratings</h2>
+                    <ul class="rating">
+                        <li class="rating-control"><i class="fa fa-star"></i></li>
+                        <li class="rating-control"><i class="fa fa-star"></i></li>
+                        <li class="rating-control"><i class="fa fa-star"></i></li>
+                        <li class="rating-control"><i class="fa fa-star"></i></li>
+                        <li class="rating-control"><i class="fa fa-star"></i></li>
+                        <li class="rating-count">0</li>
+                    </ul>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522rating%2522%253E%250A%2520%2520%253Cli%2520class%253D%2522rating-control%2522%253E%253Ci%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522rating-control%2522%253E%253Ci%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522rating-control%2522%253E%253Ci%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522rating-control%2522%253E%253Ci%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522rating-control%2522%253E%253Ci%2520class%253D%2522fa%2520fa-star%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522rating-count%2522%253E0%253C%252Fli%253E%250A%253C%252Ful%253E"
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="code-container">
+                    <h2>Show Ratings</h2>

```

Comment:
I do not understand what these numbers mean (62, 2, 62, 2, 612)

Reply:
its just to show the number of people do ratings. These numbers would be anything